client/src/lib/votecounting.test.ts

```diff
@@ -215,6 +215,49 @@ test('parsing CVRs flags when page number is set but not a number', () => {
   ])
 })
 
+test('parsing CVRs flags when page numbers is set but not an array of numbers', () => {
+  const cvr: CastVoteRecord = {
+    _ballotStyleId: '12',
+    _precinctId: '23',
+    _ballotId: 'abc',
+    _scannerId: 'scanner-1',
+    _testBallot: false,
+    // @ts-ignore
```

Comment:
`@ts-expect-error` is better here is the version of TS is recent enough to support it.

Reply:
this is not working for me.